packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -37,10 +37,8 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
 
   const popperRef = React.useRef<PopperJS>()
   const contentRef = React.useRef<HTMLElement>(null)
-  const latestPlacement = React.useRef<PopperJS.Placement>(proposedPlacement)
-  const [computedPlacement, setComputedPlacement] = React.useState<PopperJS.Placement>(
-    proposedPlacement,
-  )
+  const latestPlacement = React.useRef<PopperJS.Placement>()
```

Comment:
where this guy is used? highly likely we should remove it now

Reply:
line 118, not in the scope of the PR